skcosmo/preprocessing/flexible_scaler.py

```diff
@@ -16,17 +16,17 @@ class StandardFlexibleScaler(TransformerMixin, BaseEstimator):
     :type with_std: boolean
     :param column_wise: If True, normalize each column separately. If False, normalize the whole matrix, divided it by variaton.
     :type column_wise: boolean
-    :param tol: The tolerance for the optimization: if the variance are smaller than tol, it is considered zero.
-
+    :param rtol: The relative tolerance for the optimization: if the inequality variance/mean< rtol is satisfied, the variance is considered zero.
+    :type rtol: int or float
```

Comment:
```suggestion
    :type rtol: float
```

Reply:
int does not really make sense here, since we expect values between `1e0` and `1e-30`